Refit-Tests/InterfaceStubGenerator.cs

```diff
@@ -57,9 +57,9 @@ public void HasRefitHttpMethodAttributeSmokeTest()
             var fixture = new InterfaceStubGenerator();
 
             var input = file.GetRoot().DescendantNodes()
-                .OfType<InterfaceDeclarationSyntax>()
-                .SelectMany(i => i.Members.OfType<MethodDeclarationSyntax>())
-                .ToList();
+                            .OfType<InterfaceDeclarationSyntax>()
```

Comment:
Looking at this on my machine, are you sure your local settings aren't somehow overriding the project settings? These indents shouldn't be happening - if I do _Tools > Cleanup Code..._ then select _Default: Reformat Code_, these indents are reset to what they were before.